### PR TITLE
Document font update pitfall

### PR DESCRIPTION
**Pull request by cadadr**
If a font file associated to a font used by Qtile is updated, Qtile may crash.  The resulting logs might be somewhat confusing.  This commit documents this pitfall in three places, including the default Qtile configuration file so that users are aware of it and can avoid it.

This adresses #2656.

### CODE REVIEW

File: docs/manual/config/index.rst [hunk 1]
```diff
@@ -163,7 +163,10 @@ configuration variables that control specific aspects of Qtile's behavior:
       - dict(font='sans',
              fontsize=12,
              padding=3)
-      - Default settings for bar widgets.
+      - Default settings for bar widgets.  Note: if the font file
+        associated with the font selected here is modified while Qtile
+        is running, Qtile may segfault (for details see `issue #2656
+        <https://github.com/qtile/qtile/issues/2656>`_).
```
Comment (tych0) [inline docs/manual/config/index.rst:169]: Can we just leave this one? Doesn't feel like we need to say the same thing three times.

Comment (cadadr) [reply docs/manual/config/index.rst:169]: Sure :+1:  updated the patch.

**Comment by ramnes** [comment]
Took the rebase on myself and merged with adeccf9a56478044fb3e23d9f55fc409ff96c7bf. Thanks!

**Comment by cadadr** [comment]
You're welcome and thanks a lot!

**Pull request closed without merging**